Formula/dasm.rb

```diff
@@ -3,7 +3,7 @@ class Dasm < Formula
   homepage "https://dasm-assembler.github.io/"
   url "https://github.com/dasm-assembler/dasm/archive/2.20.14.tar.gz"
   sha256 "c66538ad8c413a4ea88525246ed9fef2bf9c9d2e36593acdf651e06635ad7497"
-  license "GPL-2.0"
+  license "GPL-2.0-or-later"
```

Comment:
```suggestion
  license "GPL-3.0-or-later"
```

Changed 20 days ago: https://github.com/dasm-assembler/dasm/commit/4f620f3b35655521236c30a69d762f7ec1fce9c0 (see `docs/sources/legal.tex`)

Many files still have the `GPL-2.0+` header

Reply:
Given they switched to `GPL-3.0`, I think it should be reflected in the headers. I opened a PR at https://github.com/dasm-assembler/dasm/pull/89 to make this change. Let's hold off on this until we get a response.

Reply:
The change to `GPL-3.0` has been reverted and all files are now `GPL-2.0` (see https://github.com/dasm-assembler/dasm/pull/89)